actions/raw_action.go

```diff
@@ -1,3 +1,28 @@
+/*
+Raw Action
+
+Allows to write some file to output image with given offset.
+Usually this is needed for bootloader stuff.
+
+Yaml syntax:
+ - action: raw
+   origin: name
+   source: filename
+   offset: bytes
+
+Mandatory properties:
+
+- origin -- reference to named file or directory.
+
+- source -- the name of file located in 'origin' to be written into the output image.
+
+Optional properties:
+
+- offset -- offset in bytes for output image file.
+It is possible to use internal templating mechanism of debos to calculate offset
```

Comment:
fwiw that's a go templating function debos implements. We should have some notes about the go templating usage somewhere else i guess

Reply:
I still need to describe the overall concept.